src/plugins/loan-channel-query-map.js

```diff
@@ -53,7 +53,7 @@ export default {
 					url: 'livestock',
 					queryParams: 'status=fundRaising&riskRating=0,5&activity=73',
 					// algoliaParams: '', // not supported pass exclude link
-					fallbackUrl: '/lend?status=fundRaising&riskRating=0,5&activity=73&kexpn=lend_filter.lend_filter_versions&kexpv=c'
+					fallbackUrl: '/lend?status=fundRaising&riskRating=0,5&activity=73'
```

Comment:
Removing the old exp activation for fallback as you can now just hit lend without the override.